### PR TITLE
fsevents-tools (new formula)

### PR DESCRIPTION
**Pull request by Hardmath123**
Add `fsevents-tools` by @ggreer (described as inotify-tools for OS X's FSEvents).

GitHub: https://github.com/ggreer/fsevents-tools
Home: http://geoff.greer.fm/fsevents/
Blog post: http://geoff.greer.fm/2015/12/25/fsevents-tools-watch-a-directory-for-changes/

Thanks! :-)

### CODE REVIEW

**Comment by bfontaine** [comment]
The test is timing out.

**Comment by Hardmath123** [comment]
Ah, the reason is that the test involves a 5-second "sleep" in a new thread (to create a "dummy" event for the program to notice). What's the timeout? We can reduce the sleep duration by quite a bit if needed.

**Comment by bfontaine** [comment]
IIRC our test limit is 60 or 90 minutes. I guess the program is waiting indefinitely.

**Comment by bfontaine** [comment]
You could try something like that:

```rb

  test do
    testfile = testpath/"test-file"

    pid = fork do
      sleep 5
      touch testfile
    end

    system "#{bin}/notifywait", "."
    Process.kill "TERM", pid
    testfile.unlink
  end
```

File: Library/Formula/fsevents-tools.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    # Make sure notifywait exits when test-file is created.
+    system "bash", "-c", "(sleep 5.0; touch test-file;) &"
```
Comment (mikemcquaid) [inline Library/Formula/fsevents-tools.rb]: Please do this with Ruby's `sleep` and `touch` instead.

**Comment by DomT4** [comment]
Closing as dead. Please feel free to resubmit with comments addressed if desired. Thank for your submission to Homebrew!

**Pull request closed without merging**